library/ssl_misc.h

```diff
@@ -2095,6 +2097,41 @@ int mbedtls_ssl_tls13_write_early_data_ext(mbedtls_ssl_context *ssl,
                                            unsigned char *buf,
                                            const unsigned char *end,
                                            size_t *out_len);
+
+#if defined(MBEDTLS_SSL_SRV_C)
```

Comment:
I do not think we need these utility functions, at least currently. It is defining new symbols that do not bring much than just accessing `early_data_status` directly.